src/app/public/modules/modal/fixtures/modal-with-focus-content.fixture.html

```diff
@@ -1,6 +1,8 @@
 <sky-modal>
   <sky-modal-header>Test</sky-modal-header>
   <sky-modal-content>
-    <button>Content</button>
+    <button style="visibility: hidden">Hidden</button>
+    <button style="display: none">No display</button>
```

Comment:
These are great. Should we have an element with 0 size too? The adapter function checks for bounds along with `visibility` and `display`.

Reply:
Oh yeah, it didn't have a test for that. This is actually to check if it is displayed as well (has a bounds and a visual offset in the viewport) I'll add one of the other display states it covers. Nice catch 👍

Reply:
Alright, so IE and Edge do not seem to support the only other option I came across `display: contents`, so covering this in the test will fail in IE and Edge. Unfortunately I had to take that out